src/components/cutscenes/OakIntroPokemonChoiceCutscene.ts

```diff
@@ -25,7 +25,10 @@ export class OakIntroPokemonChoiceCutscene<TGameStartr extends FullScreenPokemon
                 "GeneralText",
                 [
                     "Those are %%%%%%%POKE%%%%%%% Balls. They contain %%%%%%%POKEMON%%%%%%%!",
-                ]);
+                ],
+                (): void => {
+                    this.gameStarter.menuGrapher.deleteActiveMenu();
```

Comment:
This is one way to do it! However, it's a little cleaner to use `deleteOnFinish: true` in creating the `"GeneralText"` menu. Please switch to that.